Makefile

```
@@ -1,3 +1,69 @@
+IMAGE_REGISTRY ?= quay.io
+REGISTRY_NAMESPACE ?= ocs-dev
```

- can we not have this var set and let it fail when not supplied?
- I'm not sure of all other vars and at the same time this mostly doesn't fit as a default destination

---

What i have seen in most places is that we make default in the Makefile, if this is the standard followed in managed service I can switch to that model.

---

ack, what I only meant was `ocs-dev` is a bad default, since I don't have a good one, asked it to be empty.

---

I agree with @leelavg! why does the FaaS toolchain have an OCS default?

---

@nb-ohad am not sure what do we have for Faas, I can update the same here, this is just a place holder

---

We don't! keep it empty